docs/setup/installing/index.rst

```diff
@@ -0,0 +1,52 @@
+.. _installing:
+
+.. toctree::
+
+=========================================
+Installing VPP Binaries from Packages
+=========================================
+If you are simply using vpp, it can be convenient to simply install the packages.  The instructions below
+will describe how pull, install and install the VPP packages.
+
+.. toctree::
+
+Package Descriptions
+----------------------------------------
+The following is a brief description of the packages to be installed with VPP.
+
+.. toctree::
+
+   packages
+
+Installing on Ubuntu
+----------------------------------------
+The following are instructions on how to install VPP on Ubuntu.
+
+.. toctree::
+
+   ubuntu
+
+Installing on Centos
+----------------------------------------
+The following are instructions on how to install VPP on Centos.
+
+.. toctree::
+
+   centos
+
+Installing on openSUSE
+----------------------------------------
+The following are instructions on how to install VPP on openSUSE.
+
+.. toctree::
+
+   opensuse
+
+Downloading the jvpp jar
```

Comment:
This doesn't tell me much about what the jvpp jar is  - or why I would want to install it.

Reply:
This should go in the API Section. jvpp.jar is the VPP api.